### ISSUE TITLE
Containerlab directory no longer removed with latest update of CL

### ISSUE THREAD

**Issue by JulioPDX** [ipspace/netsim-tools#218]
```
(venv) juliopdx@containerlab:~/repos/cl-nst-demo/priv$ netlab version
netsim-tools version 1.1.3-post1
(venv) juliopdx@containerlab:~/repos/cl-nst-demo/priv$ sudo containerlab version

                           _                   _       _     
                 _        (_)                 | |     | |    
 ____ ___  ____ | |_  ____ _ ____   ____  ____| | ____| | _  
/ ___) _ \|  _ \|  _)/ _  | |  _ \ / _  )/ ___) |/ _  | || \ 
( (__| |_|| | | | |_( ( | | | | | ( (/ /| |   | ( ( | | |_) )
\____)___/|_| |_|\___)_||_|_|_| |_|\____)_|   |_|\_||_|____/ 

    version: 0.24.0
     commit: c6ff2176
       date: 2022-02-22T13:50:23Z
     source: https://github.com/srl-labs/containerlab
 rel. notes: https://containerlab.srlinux.dev/rn/0.24/
```

When trying to "down" the topology, the following error is seen.

```
(venv) juliopdx@containerlab:~/repos/cl-nst-demo/priv$ time netlab down server.yml 

Step 1: Checking virtualization provider installation
============================================================
.. all tests succeeded, moving on


Step 2: stopping the lab
============================================================
INFO[0000] Parsing & checking topology file: clab.yml   
ERRO[0000] error deleting lab directory: unlinkat /home/juliopdx/repos/cl-nst-demo/priv/clab-ebgp/B/flash: directory not empty 
INFO[0000] Destroying lab: ebgp                         
INFO[0001] Removed container: clab-ebgp-B               
INFO[0002] Removed container: clab-ebgp-A               
INFO[0002] Removed container: clab-ebgp-E               
INFO[0002] Removed container: clab-ebgp-F               
INFO[0002] Removed container: clab-ebgp-C               
INFO[0002] Removed container: clab-ebgp-G               
INFO[0002] Removed container: clab-ebgp-D               
INFO[0002] Removing containerlab host entries from /etc/hosts file 
```

Thank you as always!

**Comment by ipspace** [comment]
Considering that all we're doing is executing **sudo containerlab destroy -t clab.yml --cleanup**, and that the INFO messages are generated by **containerlab**, I'd say it's their bug, not ours ;)

Thanks for reporting it -- @jbemmel, do you have a direct link to the containerlab team, otherwise I'll raise an issue in their GitHub repo.

**Comment by JulioPDX** [comment]
> Considering that all we're doing is executing **sudo containerlab destroy -t clab.yml --cleanup**, and that the INFO messages are generated by **containerlab**, I'd say it's their bug, not ours ;)
> 
> Thanks for reporting it -- @jbemmel, do you have a direct link to the containerlab team, otherwise I'll raise an issue in their GitHub repo.

More than happy to open an issue on the CL github repo. Thank you, Ivan!

**Comment by jbemmel** [comment]
Fixed in 0.24.1 https://github.com/srl-labs/containerlab/issues/799

**Comment by ipspace** [comment]
That was fast. Thanks a million!
